.gitignore

```diff
@@ -4,4 +4,7 @@ credentials.json
 docs/meta/env
 config-connector/tests/testcases/environments.yaml
 .DS_Store
-.vscode
\ No newline at end of file
+.vscode
+
+# tf lock file
+.terraform.lock.hcl
```

Comment:
Nit: can you confirm if there's a newline at the end?

Reply:
Yup, I added one in my change.